config/locales/emails.fr.yml

```diff
@@ -0,0 +1,60 @@
+fr:
+  emails:
+    event_acceptance:
+      subject: "Votre demande pour %{conference}"
+      greeting: "Cher %{forename},"
+      accepted_event: |
+        Nous sommes très heureux de vous annoncer que nous avons accepté votre proposition
+        
+        %{event}
+      confirmation: |
+        Veuillez re-confirmer rapidement que vous serez présent et ferez bien votre
+        conférence, en utilisant le lien suivant :
+
+        %{link}
+      check_data: |
+        Pendant que vous y êtes : Après confirmation avec ce lien, vous serez
+        identifé·e dans le système de planning de conférence. Prenez le temps
+        de vérifier vous information à ce moment. Les informations seront
+        utilisées dans le programme officiel de la conférence. Vous pouvez
+        aussi spécifier vos disponibilités pour d'autres horaires. Nous allons
+        prendre ces horaires en compte pour le planning final.
+      goodbye: |
+        Cordialement,
+
+        L'équipe de %{conference}
```

Comment:
Maybe remove "de"?

L'équipe de Évènement du siècle → :confused: 
L'équipe de l'Évènement du siècle → :grin: 
L'équipe Évènement du siècle → :relieved: 

L'équipe de FrOSCon → :grin:
L'équipe de l'FrOSCon →  :confused: 
L'équipe FrOSCon → :relieved:

Reply:
Ok, that's way better